htdocs/editlog.php

```diff
@@ -61,382 +57,470 @@
-                        tpl_set_var('logtext', htmlspecialchars($log_text, ENT_COMPAT, 'UTF-8'), true);
-                    else
-                        tpl_set_var('logtext', $log_text);
-
-                    // Text / normal HTML / HTML editor
-                    tpl_set_var('use_tinymce', (($descMode == 3) ? 1 : 0));
-
-                    $headers = tpl_get_var('htmlheaders') . "\n";
-                    if ($descMode == 1)
-                        tpl_set_var('descMode', 1);
-                    else if ($descMode == 2)
+                //set template vars
+                tpl_set_var('cachename', htmlspecialchars($cache_name, ENT_COMPAT, 'UTF-8'));
+                tpl_set_var('logtypeoptions', $logtypeoptions);
+                tpl_set_var('logday', htmlspecialchars($log_date_day, ENT_COMPAT, 'UTF-8'));
+                tpl_set_var('logmonth', htmlspecialchars($log_date_month, ENT_COMPAT, 'UTF-8'));
+                tpl_set_var('logyear', htmlspecialchars($log_date_year, ENT_COMPAT, 'UTF-8'));
+                tpl_set_var('loghour', htmlspecialchars($log_time_hour, ENT_COMPAT, 'UTF-8'));
+                tpl_set_var('logminute', htmlspecialchars($log_time_minute, ENT_COMPAT, 'UTF-8'));
+                tpl_set_var('cachename', htmlspecialchars($cache_name, ENT_COMPAT, 'UTF-8'));
+                tpl_set_var('cacheid', $log_record['cache_id']);
+                tpl_set_var('reset', $reset);  // obsolete
+                tpl_set_var('submit', $submit);
+                tpl_set_var('logid', $log_id);
+                tpl_set_var('date_message', !$date_ok ? $date_message : '');
+
+                if ($descMode != 1) {
+                    tpl_set_var('logtext', htmlspecialchars($log_text, ENT_COMPAT, 'UTF-8'), true);
+                } else {
+                    tpl_set_var('logtext', $log_text);
+                }
+
+                // Text / normal HTML / HTML editor
+                tpl_set_var('use_tinymce', (($descMode == 3) ? 1 : 0));
+
+                $headers = tpl_get_var('htmlheaders') . "\n";
+                if ($descMode == 1) {
+                    tpl_set_var('descMode', 1);
+                } else {
+                    if ($descMode == 2) {
                         tpl_set_var('descMode', 2);
-                    else
-                    {
+                    } else {
                         // TinyMCE
                         $headers .= '<script language="javascript" type="text/javascript" src="resource2/tinymce/tiny_mce_gzip.js"></script>' . "\n";
-                        $headers .= '<script language="javascript" type="text/javascript" src="resource2/tinymce/config/log.js.php?logid=0&lang='.strtolower($locale).'"></script>' . "\n";
+                        $headers .= '<script language="javascript" type="text/javascript" src="resource2/tinymce/config/log.js.php?logid=0&lang=' . strtolower(
+                                $locale
+                            ) . '"></script>' . "\n";
```

Comment:
Please remove an indent from the lines 461 and 462, as to PSR-2 rule: _Argument lists MAY be split across multiple lines, where each subsequent line is indented **once**._